src/main/pages/che-7/extensions/che4z-installing.adoc

```diff
@@ -25,9 +25,9 @@ This page describes how to install the Eclipse Che4z basic stack.
 
 . In a web browser, load `++https://++__<YOUR_CHE_HOST>__/f?url=__<DEV_FILE_LOCATION>__`.
 +
-* For the Eclipse Che4z basic stack, use the link:https://github.com/eclipse/che-che4z/raw/1.0.0/mainframe-basic-stack.yaml[`mainframe-basic-stack.yaml`] devfile.
+* For the Eclipse Che4z basic stack, use the devfile `https://github.com/eclipse/che-che4z/raw/1.0.0/mainframe-basic-stack.yaml`
```

Comment:
Reason for this? We try to avoid naked URLs -- why is this needed? Also, using the `link:` notation is good practice for live links. As for word order, "use the foo devfile" is IMO better than "use the devfile foo".

Reply:
We provide the URL so that the user can copy and paste it, not so the user can click the link and download the file. Surely a codeblock is preferable to a link for this purpose?

Reply:
Zeb is right - we don't want users to click on the link, we want them only to copy the string with URL.

Reply:
Ok, in that case, the link should not be live.
```suggestion
* For the Eclipse Che4z basic stack, use the `++https://github.com/eclipse/che-che4z/raw/1.0.0/mainframe-basic-stack.yaml++` devfile.
```